docs/source/cuda-reference/host.rst

```diff
@@ -109,12 +109,16 @@ invoking any of the driver API. This can be useful for:
 .. autofunction:: numba.cuda.compile_ptx
 
 
-If PTX for the compute capability of the current device is required, the
-``compile_ptx_for_current_device`` function can be used:
+The configuration variable ``CUDA_DEFAULT_PTX_CC`` can be set to control the
```

Comment:
Should that be `NUMBA_CUDA_DEFAULT_PTX_CC` ?

Reply:
The environment variable is `NUMBA_CUDA_DEFAULT_PTX_CC`, but the variable in `numba.config` is called `CUDA_DEFAULT_PTX_CC`, which I refer to as a "configuration variable" - is this terminology OK / understandable?

Reply:
It's clear to me, however, my concern is, as a new user, would it be clear from this how to set it? `numba.config` use isn't particularly well defined and the semantics of use are nuanced, whereas setting an env var is well known?

Reply:
OK - changed to refer to the environment variable.

Reply:
Great, thanks!